bip-emilengler-replacing-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Replacing the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
```

Comment:
A definition is needed. Particularly one that should be inclusive of Lightning or other L2 networks.

Reply:
I added a note for the ''address'' part, hop it is sufficient

Reply:
I don't see a definition in the current draft either...

Someone who doesn't know "address" needs to be able to understand this BIP.

Reply:
"A bitcoin invoice address is a human-readable encoding of a scriptPubKey, for example bech32 addresses BIP173 or base58 legacy addresses"?

Reply:
No, that's too specific to technical details (and excludes L2 stuff), and neglects to cover the intent/purpose of the invoice address.

Maybe something like "A bitcoin invoice [address] is a string of characters that can be used to indicate the intended recipient and purpose of a transaction." ?

Reply:
Ok, good to know
I will use @luke-jr definition.
It should contain that it in affiliated to the recipient in some way.

Reply:
Done